mp/game/momentum/momentum.fgd

```diff
@@ -7,6 +7,13 @@
 // -----------------------------------------------------------------------
 @SolidClass base(Targetname, EnableDisable, Toggle) = func_nogrenades : "Rockets and grenades will not detonate inside this area." 
 [
+	fizzle_type(choices) : "Stickybomb prevention method" : 0 =

```

Comment:
```suggestion
    fizzle_type(choices) : "Stickybomb prevention method" : 0 =
```

Use space indenting here instead of tabs. Everywhere else as well.